adobecom/milo
# MWPW-146582: a11y aria attributes

## Description
This PR fixes the a11y issue where the video button is not using permitted aria attributes.

## Related Issue
Resolves: [MWPW-145514](https://jira.corp.adobe.com/browse/MWPW-145514)

## Testing instructions
1. go to the test url
2. open [axe DevTools](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd)
3. scan the entire page
<img width="1546" alt="Screenshot 2024-04-17 at 10 24 02" src="https://github.com/adobecom/milo/assets/146744221/70e14fde-47e8-4dc6-b391-8cd4c585e45f">
4. using the _before url_ you should see the _"Elements must only use permitted ARIA attributes"_ issue. On the _after url_, that issue should not be present anymore.

## Screenshots:
**Before**
![image](https://github.com/adobecom/milo/assets/146744221/f36863f9-9b0b-442d-be60-228a6f066855)

**After**
![image](https://github.com/adobecom/milo/assets/146744221/5100801d-d3cd-4331-b450-8433157bce8d)

## Test URLs
**Acrobat:**
- Before: https://www.stage.adobe.com/acrobat/online/sign-pdf.html
- After: https://www.stage.adobe.com/acrobat/online/sign-pdf.html?milolibs=mwpw-145414-a11y-aria-attributes--milo--robert-bogos

**BACOM:**
- Before: https://main--bacom--adobecom.hlx.page/products/real-time-customer-data-platform/rtcdp
- After: https://main--bacom--adobecom.hlx.page/products/real-time-customer-data-platform/rtcdp?milolibs=mwpw-145414-a11y-aria-attributes--milo--robert-bogos

**CC:**
- Before: https://main--cc--adobecom.hlx.page/drafts/rbogos/default?martech=off
- After: https://main--cc--adobecom.hlx.page/drafts/rbogos/default?martech=off&milolibs=mwpw-145414-a11y-aria-attributes--milo--robert-bogos

**Homepage:**
- Before: https://main--homepage--adobecom.hlx.page/homepage/index-loggedout?
- After: https://main--homepage--adobecom.hlx.page/homepage/index-loggedout?milolibs=mwpw-145414-a11y-aria-attributes--milo--robert-bogos

**Blog:**
- Before: https://main--blog--adobecom.hlx.page/
- After: https://main--blog--adobecom.hlx.page/?milolibs=mwpw-145414-a11y-aria-attributes--milo--robert-bogos

**Milo:**
- Before: https://main--milo--adobecom.hlx.page/drafts/mathuria/genericvideobtn/demovideomodal?martech=off
- After: https://mwpw-145414-a11y-aria-attributes--milo--robert-bogos.hlx.page/drafts/mathuria/genericvideobtn/demovideomodal?martech=off
